WordPress/Classes/Services/CommentService.m

```diff
@@ -1132,6 +1132,39 @@ - (Comment *)findCommentWithID:(NSNumber *)commentID fromPost:(ReaderPost *)post
 #pragma mark - Transformations
 
 - (void)updateComment:(Comment *)comment withRemoteComment:(RemoteComment *)remoteComment
+{
+    comment.commentID = remoteComment.commentID;
+    comment.author = remoteComment.author;
+    comment.author_email = remoteComment.authorEmail;
+    comment.author_url = remoteComment.authorUrl;
+    comment.authorAvatarURL = remoteComment.authorAvatarURL;
+    // Ref: https://github.com/wordpress-mobile/WordPressKit-iOS/pull/413
+    // Use raw_content for blog comments so
+    // This is a temporary mixmatch of properties for a frozen branch bug fix that we'll sort out properly in develop.
+    // - @aerych
+    comment.content = remoteComment.rawContent;
+    comment.dateCreated = remoteComment.date;
+    comment.link = remoteComment.link;
+    comment.parentID = remoteComment.parentID;
+    comment.postID = remoteComment.postID;
+    comment.postTitle = remoteComment.postTitle;
+    comment.status = remoteComment.status;
+    comment.type = remoteComment.type;
+    comment.isLiked = remoteComment.isLiked;
+    comment.likeCount = remoteComment.likeCount;
+    comment.canModerate = remoteComment.canModerate;
+
+    // if the post for the comment is not set, check if that post is already stored and associate them
+    if (!comment.post) {
+        PostService *postService = [[PostService alloc] initWithManagedObjectContext:self.managedObjectContext];
+        comment.post = [postService findPostWithID:comment.postID inBlog:comment.blog];
+    }
+}
+
+// Temporary duplication of [updateComment:withRemoteComment:] to address an issue in a frozen branch
+// We'll sort it out properly in develop.
+// - @aerych
+- (void)updateReaderComment:(Comment *)comment withRemoteComment:(RemoteComment *)remoteComment
```

Comment:
I know this is a quick temp fix, but couldn't this just be:

```
[self updateComment:comment withRemoteComment:remoteComment];
comment.content = remoteComment.content;
```

Reply:
Ah it totally could have been. 🤦   Oh wellz.  Temp code ftw I guess.